<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>324</width>
    <height>372</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>324</width>
    <height>74</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>324</width>
    <height>372</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>WordAssisant</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Sources.qrc">
    <normaloff>:/image/scanner.png</normaloff>:/image/scanner.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="StripListView" name="centralwidget">
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBarTop">
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>toolBarTop</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenWordFile"/>
   <addaction name="actionCreateWordFile"/>
   <addaction name="separator"/>
   <addaction name="actionCaptureText"/>
   <addaction name="actionCapturePicture"/>
   <addaction name="actionPasteInWord"/>
   <addaction name="separator"/>
   <addaction name="actionSetting"/>
  </widget>
  <widget class="QToolBar" name="toolBarBottom">
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>toolBarBottom</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomToolBarArea</set>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAddStrip"/>
   <addaction name="actionSubstractStrip"/>
   <addaction name="actionSave"/>
  </widget>
  <action name="actionOpenWordFile">
   <property name="icon">
    <iconset resource="Sources.qrc">
     <normaloff>:/image/open.png</normaloff>:/image/open.png</iconset>
   </property>
   <property name="text">
    <string>Open WORD File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionCreateWordFile">
   <property name="icon">
    <iconset resource="Sources.qrc">
     <normaloff>:/image/new.png</normaloff>:/image/new.png</iconset>
   </property>
   <property name="text">
    <string>Create WORD File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSetting">
   <property name="icon">
    <iconset resource="Sources.qrc">
     <normaloff>:/image/setting.png</normaloff>:/image/setting.png</iconset>
   </property>
   <property name="text">
    <string>Setting</string>
   </property>
   <property name="toolTip">
    <string>Setting</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F4</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="Sources.qrc">
     <normaloff>:/image/quit.png</normaloff>:/image/quit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+F4</string>
   </property>
  </action>
  <action name="actionCaptureText">
   <property name="icon">
    <iconset resource="Sources.qrc">
     <normaloff>:/image/text.png</normaloff>:/image/text.png</iconset>
   </property>
   <property name="text">
    <string>Capture Text</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionCapturePicture">
   <property name="icon">
    <iconset resource="Sources.qrc">
     <normaloff>:/image/picture.png</normaloff>:/image/picture.png</iconset>
   </property>
   <property name="text">
    <string>Capture Picture</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actionPasteInWord">
   <property name="icon">
    <iconset resource="Sources.qrc">
     <normaloff>:/image/paste.png</normaloff>:/image/paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste In Word</string>
   </property>
   <property name="shortcut">
    <string>F4</string>
   </property>
  </action>
  <action name="actionMainMenu">
   <property name="icon">
    <iconset resource="Sources.qrc">
     <normaloff>:/image/MainMenu.png</normaloff>:/image/MainMenu.png</iconset>
   </property>
   <property name="text">
    <string>MainMenu</string>
   </property>
   <property name="toolTip">
    <string>Main Menu</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionAddStrip">
   <property name="icon">
    <iconset resource="Sources.qrc">
     <normaloff>:/image/add.png</normaloff>:/image/add.png</iconset>
   </property>
   <property name="text">
    <string>AddStrip</string>
   </property>
   <property name="toolTip">
    <string>Add Strip</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+=</string>
   </property>
  </action>
  <action name="actionSubstractStrip">
   <property name="icon">
    <iconset resource="Sources.qrc">
     <normaloff>:/image/delete.png</normaloff>:/image/delete.png</iconset>
   </property>
   <property name="text">
    <string>SubstractStrip</string>
   </property>
   <property name="toolTip">
    <string>Substract Strip</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="Sources.qrc">
     <normaloff>:/image/save.png</normaloff>:/image/save.png</iconset>
   </property>
   <property name="text">
    <string>actionSave</string>
   </property>
   <property name="toolTip">
    <string>Save Strip</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StripListView</class>
   <extends>QWidget</extends>
   <header>striplistview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Sources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionMainMenu</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>153</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
